README.md

```diff
@@ -829,6 +831,31 @@ Maximum Trasnmission Unit size for frames received and sent on the specified
 interface. Valid value is an integer.
 
 --
+### Type: `network_trunk`
```

Comment:
Please move the attributes detail under the NetDev Stdlib section:

https://github.com/HelenCampbell/cisco-network-puppet-module/blob/network_trunk/README.md#netdev-stdlib-resource-type-details